index.html

```diff
@@ -21,27 +21,218 @@
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                
+                
+                
+                
+                
+            </div>
+        </section>
+        <section>
+            <div class="flex-container">
+                <div id="about-title">
+                    <div id="about-box" class="font-normal">
+                        <p>About me</p>
+                    </div>
+                    <div id="about-text" class="font-normal">
+                        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                            don’t hesitate to contact me.</p>
+                    </div>
+                    <div class="btn-div">
+                        <button class="btn">Get my resume</button>
+                    </div>
+                </div>
```

Comment:
- Kindly add the appropriate background image to the About section, reducing the size of the text, and reducing the size of `get my resume`  and  it will make your design stick more to the Figma design as required in the project :+1: 

## Your design 

![image](https://github.com/Armynerh/personal_portfolio/assets/95297251/a614c936-eea1-4d71-a158-b0a778f7f8e5)

## Figma design 

![image](https://github.com/Armynerh/personal_portfolio/assets/95297251/8ceaa015-c9ee-4022-b606-eb3a136f0ee4)